packages/ra-core/src/controller/useCreateController.ts

```diff
@@ -39,8 +39,8 @@ export interface CreateProps {
     hasEdit?: boolean;
     hasList?: boolean;
     hasShow?: boolean;
-    location: Location;
-    match: Match;
+    location?: Location; // deprecated
```

Comment:
it's not deprecated, it's ignored. react-router still injects these props.

Reply:
Would you like me to remove it from the interface instead?

Reply:
no, just remove the `// deprecated` comment